mobx/lib/mobx.dart

```diff
@@ -29,8 +29,8 @@
 library mobx;
 
 export 'package:mobx/src/api/action.dart';
-export 'package:mobx/src/api/annotations.dart'
-    show action, computed, observable, store;
+export 'package:mobx/src/api/annotations.dart';
+//    show action, computed, observable, StoreConfig;
```

Comment:
We need to uncomment this line. The reason we have this is not to expose `Make**` classes, hence the "show statement"

Reply:
Sorry, I forgot this one.